### PR TITLE
feat(@angular-devkit/build-angular): update webpack dependencies

### PR DESCRIPTION
**Pull request by clydin**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7778,13 +7878,20 @@ webpack-sources@^1.0.1, webpack-sources@^1.1.0:
     source-list-map "^2.0.0"
     source-map "~0.6.1"
 
+webpack-sources@^1.2.0:
```
Comment (alan-agius4) [inline yarn.lock]: Nice, this should improve the CI build as well. As people are claiming good improvements on webpack when having sourcemaps enabled.

Comment (filipesilva) [reply yarn.lock]: Would be cool to benchmark it the before and after with #12022. Only after the CI error is addressed though.

**Comment by ralshammrei2** [comment]
#12038

**Comment by clydin** [comment]
Covered by #12185

**Pull request closed without merging**